packages/angular/cli/package.json

```diff
@@ -41,6 +41,9 @@
     "symbol-observable": "1.2.0"
   },
   "ng-update": {
-    "migrations": "@schematics/angular/migrations/migration-collection.json"
+    "migrations": "@schematics/angular/migrations/migration-collection.json",
+    "packageGroup": {
+      "@angular/cli": "0.0.0"
```

Comment:
Shouldn’t this package group contain the packages to updates? Aka for the cli shouldn’t it be @angular-devkit/build-angular and @angular-devkit/ng-packagr?

Reply:
It's all done in the `build.ts` actually.

Reply:
I'll remove this line. it's basically useless.